Remove paddlepaddle dep

tabossert opened the pull request:
We have created a custom wheel for aarch64 of paddlepaddle, however pypi requires a unique name of packages so we had to name it unstructured.paddlepaddle.  When it’s a dep for paddleocr it doesn’t work on aarch64.

We will take care of installing these packages in unstructured and unstructured-api in Dockerfiles and install scripts

tabossert committed:
Remove paddlepaddle dep

We have created a custom wheel for aarch64 of paddlepaddle, however pypi requires a unique name of packages so we had to name it unstructured.paddlepaddle.  When it’s a dep for paddleocr it doesn’t work on aarch64.

We will take care of installing these packages in unstructured and unstructured-api

yuming-long commented:
btw could you remove this as well https://github.com/Unstructured-IO/unstructured.PaddleOCR/blob/main/requirements.txt#L20

tabossert committed:
remove paddlepaddle dep

This needs to be removed so that we can install paddlepaddle wheel based on architecture, as aarch64 doesn’t have an official paddlepaddle wheel.

We install paddlepaddle/unstructured.paddlepaddle and paddleocr as part of build
